FREQ_CONTROL : 1


mode : 0